### PR TITLE
Updated all SFC to be arrow functions rather than named functions.

### PR DESCRIPTION
**Pull request by thetornadotitan**
Updated all SFC to be arrow functions rather than named functions to match desired stylings.

### CODE REVIEW

**Comment by thetornadotitan** [comment]
[SFCs for main pages are not arrow functions and should be.](https://trello.com/c/MJCnGETO/55-sfcs-for-main-pages-are-not-arrow-functions-and-should-be)

File: src/components/contributors/index.js [hunk 1]
```diff
@@ -11,10 +11,12 @@ const styles = StyleSheet.create({
   }
 });
 
-export default function Contributors() {
-  return (
+const Contributors = () => {
+  (
     <View style={styles.content}>
```
Comment (vrumger) [inline src/components/contributors/index.js]: Did you remove the `return` on purpose?

**Review by alexander-alegre — APPROVED** [review]
Code changes look good and follow the standards.

File: src/components/contributors/index.js (hunk 1 above)
Comment (thetornadotitan) [reply src/components/contributors/index.js]: > Did you remove the `return` on purpose?

I did but as CJ is showing I didn't do it quite right lol.

Comment (w3cj) [reply src/components/contributors/index.js]: Please remove curly braces to be able to leave off return. Applies to all other functions as well.

**Review by w3cj — CHANGES_REQUESTED** [review]
Please see comments above and update.

**Review by w3cj — APPROVED** [review]

File: src/components/contributors/index.js (hunk 1 above)
Comment (thetornadotitan) [reply src/components/contributors/index.js]: Fixed incorrect curly brace inclusion.

**Pull request merged**